arangod/Cluster/ClusterEdgeCursor.h

```diff
@@ -52,13 +52,17 @@ class ClusterEdgeCursor : public graph::EdgeCursor {
 
   void readAll(EdgeCursor::Callback const& callback) override;
 
+  /// @brief number of HTTP requests performed.
+  size_t httpRequests() const override { return _httpRequests; }
+
  private:
   std::vector<arangodb::velocypack::Slice> _edgeList;
 
   size_t _position;
   CollectionNameResolver const* _resolver;
   arangodb::graph::BaseOptions* _opts;
   arangodb::graph::ClusterTraverserCache* _cache;
+  size_t _httpRequests;
```

Comment:
@jsteemann Maybe I've just missed it, but I don't see where you're increasing `_httpRequests`. In the constructors `fetchEdgesFromEngines` from `Cluster/ClusterMethods.cpp` is called, which seems to make one request per engine.

Reply:
Good catch!
I think the instance variable in `ClusterEdgeCursor.h` is currently not increased, so I am counting only requests for vertices but not for edges.
Will add it!